### PR TITLE
8165 - Updated colors in colorpicker

### PR DESCRIPTION
**Pull request by max-hizon**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This pull request will update the color palette used in colorpicker

**Related github/jira issue (required)**:
Closes: https://github.com/infor-design/enterprise/issues/8165

**Steps necessary to review your pull request (required)**:
1. Go to http://localhost:4000/components/colorpicker/example-index.html
2. Click the color picker drop down
3. See that the color options are now updated to match current color palette. Use https://design.infor.com/code/ids-enterprise/latest/demo/components/colors/example-index?theme=uplift&variant=light as reference

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Looks good on the color picker changes but looks like the CHANGELOG and maybe contextual action panel are deleted/or changed by accident?

**Comment by max-hizon** [comment]
> Looks good on the color picker changes but looks like the CHANGELOG and maybe contextual action panel are deleted/or changed by accident?

I think I must've accidentally worked on a different branch while using this one. Will fix.

**Comment by max-hizon** [comment]
@tmcconechy After reviewing the code it seems for some reason when I edited CHANGELOG.md and contextual action panel, the edit made it so it erased its contents and replaced it with the same code along with the changes I made. For contextual action panel, it seems to have just deleted the code and replaced it with the same.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
 - `[Datagrid]` Adjusted hover styling for search and expand buttons. ([#8078](https://github.com/infor-design/enterprise/issues/8078))
 - `[Datagrid]` Fixed misalignment on date cells when selected. ([#8021](https://github.com/infor-design/enterprise/issues/8021))
 - `[Datagrid]` Fixed an issue where the new row did not display an error tooltip on the first cell. ([#8071](https://github.com/infor-design/enterprise/issues/8071))
 - `[Fileupload]` Added condition to not allow for input clearing for readonly and disabled. ([#8024](https://github.com/infor-design/enterprise/issues/8024))
 - `[Modal]` Adjusted modal title spacing to avoid icon from cropping. ([#8031](https://github.com/infor-design/enterprise/issues/8031))
 - `[Page-Patterns]` Fixed background on hovered selected tab. ([#8088](https://github.com/infor-design/enterprise/issues/8088))
 - `[Popupmenu]` Added fix for icon size in new. ([#8175](https://github.com/infor-design/enterprise/issues/8175))
-- `[Tabs-Module]` Removed padding in embedded mode. ([#8060](https://github.com/infor-design/enterprise/issues/8060))
```
Comment (yohannahbautista) [inline docs/CHANGELOG.md:18]: Unless this was removed to fix sorting or it's a duplicate, please undo this

**Review by tmcconechy — APPROVED** [review]
Thanks @max-hizon no problem. I do that too sometimes. Just flagging it. (one tip is i like to review my own PR when i first put it in).

Cheers 👍🏻 Nice Work here

**Review by jbrcna — APPROVED** [review]
nice work @max-hizon . are you a dev now?
image

**Review by yohannahbautista — APPROVED** [review]

**Review by glenlieorillo — APPROVED** [review]
image

image

**Pull request merged**